### PR TITLE
Changed the class name to bookfineUI

### PR DESCRIPTION
**Pull request by abhimaniac4**

### CODE REVIEW

File: library/returnBook/ReturnBookUI.java [hunk 1]
```diff
@@ -4,17 +4,17 @@
 
 public class ReturnBookUI {
```
Comment (Lakshansm90) [inline library/returnBook/ReturnBookUI.java]: Enum names are to start with an uppercase letter and to be in CamelBack
Ex: private enum **ControlState** {INITIALISED, READY, RESTRICTED};

**Review by Lakshansm90 — CHANGES_REQUESTED** [review]
Enum names are to start with an uppercase letter and to be in CamelBack
Ex: private enum **ControlState** {INITIALISED, READY, RESTRICTED};

File: library/returnBook/ReturnBookUI.java (hunk 1 above)
Comment (abhimaniac4) [reply library/returnBook/ReturnBookUI.java]: Ok Noted. Will make changes

**Comment by abhimaniac4** [comment]
Requesting @iamharisali to mediate the commits and ask the reviewer Lakshan to check for any changes that need to be done.

File: library/payfine/PayFineUI.java [hunk 2]
```diff
@@ -2,25 +2,25 @@
 import java.util.Scanner;
 
 
-public class PayFineUI {
+public class PayFineUi {
 
 
-	public static enum uI_sTaTe { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
+	public static enum UiState { INITIALISED, READY, PAYING, COMPLETED, CANCELLED };
```
Comment (Lakshansm90) [inline library/payfine/PayFineUI.java:8]: Put the comments

File: library/payfine/PayFineUI.java [hunk 3]
```diff
@@ -2,25 +2,25 @@
...
 
 	private pAY_fINE_cONTROL CoNtRoL;
 	private Scanner input;
-	private uI_sTaTe StAtE;
+	private UiState StAtE;
 
 	
-	public PayFineUI(pAY_fINE_cONTROL control) {
+	public PayFineUi(pAY_fINE_cONTROL control) {
```
Comment (Lakshansm90) [inline library/payfine/PayFineUI.java]: what's the purpose of the method, could please put the comment  on me method

File: library/payfine/pAY_fINE_cONTROL.java [hunk 4]
```diff
@@ -2,65 +2,65 @@
 import library.entities.Library;
 import library.entities.Member;
 
-public class pAY_fINE_cONTROL {
+public class PayFineControl {
```
Comment (Lakshansm90) [inline library/payfine/pAY_fINE_cONTROL.java:15]: please put on comment on the method and enums

**Comment by Lakshansm90** [comment]
Hi @iamharisali ,
Could you please ask @abhimaniac4 to check the comments and resolved the conflict.
Thanks,
Lakshan

**Comment by Lakshansm90** [comment]
Hi @iamharisali ,
I have resolved the conflict. Could you please ask @abhimaniac4 to do the other work.
Thanks,
Lakshan

File: library/payfine/PayFineUI.java (hunk 2 above)
Comment (abhimaniac4) [reply library/payfine/PayFineUI.java:8]: Will do

File: library/payfine/PayFineUI.java (hunk 3 above)
Comment (abhimaniac4) [reply library/payfine/PayFineUI.java]: Yes I will do it

**Review by abhimaniac4 — COMMENTED** [review]
Changes were made which were requested by @Lakshansm90

**Comment by abhimaniac4** [comment]
Requesting @iamharisali to ask @Lakshansm90 to review the commits I have added. Thank you

**Comment by iamharisali** [comment]
@Lakshansm90 Please review the changes made by @abhimaniac4

**Review by Lakshansm90 — APPROVED** [review]

**Comment by abhimaniac4** [comment]
Code is approved by @Lakshansm90

**Pull request merged**